app/components/elements/forms/checkbox_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="form-check">
+<div class="form-check mb-4">
```

Comment:
Makes the vertical spacing between form components look closer to the designs to my eyes.

Reply:
Wondering if this should be passed into the component instead of hard-coded. However, I'm willing to let it ride for now.

Reply:
I had the same thought.